logstash.yaml

```diff
@@ -69,6 +69,7 @@ pipeline:
     runs: |
       echo "gem 'fugit', '1.11.1'" >> Gemfile.template
       echo "gem 'rexml', '3.3.6'" >> Gemfile.template
+      echo "gem 'puma', '6.4.3'" >> Gemfile.template
```

Comment:
given we are bumping a dependency and want to build the package again, I think we should bump the epoch too.

Reply:
Thx for catching that